libs/cardano-data/src/Data/Orphans.hs

```diff
@@ -0,0 +1,9 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Data.Orphans () where
```

Comment:
That module and instance should be removed. As per my other comment. `cardano-binary` and `cardano-ledger-binary` should not be mixed